rhizome/postgres/bin/metrics-collector

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require "time"
+require "json"
+require "shellwords"
+
+require_relative "../../common/lib/util"
+
+if ARGV.count != 2
+  fail "Wrong number of arguments. Expected 2, Given #{ARGV.count}"
+end
+
+metrics_dir = ARGV[0]
+q_metrics_dir = metrics_dir.shellescape
+metrics_config_path = ARGV[1]
+
+fail "Metrics config file does not exist: #{metrics_config_path}" unless File.exist?(metrics_config_path)
+fail "Metrics directory does not exist: #{metrics_dir}" unless Dir.exist?(metrics_dir)
```

Comment:
In which case these wouldn't exist? I guess, maybe caller would send wrong parameter. But if we hard-code them, then no need for these checks.

Reply:
These should always exist, just added these checks to catch misconfigs easier